converters/avg_position_converter.py

```diff
@@ -0,0 +1,22 @@
+from typing import Literal
+
+import pandas as pd
+
+
+def convert_get_avgPosition_to_df(get_avgPosition_result: list[dict], period: Literal['m', 't']) -> pd.DataFrame:
+    """Преобразует данные о средней позиции в DataFrame."""
+    df = pd.DataFrame(get_avgPosition_result)
+    df = df.assign(avgPosition=df['avgPosition'].apply(lambda x: x['current']))
+    df = df[['nmId', 'avgPosition']]
```

Comment:
Не очень хорошим тоном является перезаписывание переменной результатом обработки самой самой. Лучше сделать что-то в духе
```suggestion
    df = pd.DataFrame(get_avgPosition_result)
    assigned_df = df.assign(avgPosition=df['avgPosition'].apply(lambda x: x['current']))
    cleaned_df = df[['nmId', 'avgPosition']]
```